regolith/helpers/a_proposalhelper.py

```diff
@@ -132,45 +137,63 @@ def db_updater(self):
                 "This entry appears to already exist in the collection")
         else:
             pdoc = {}
-
+        pdoc.update({'_id': key})
+        if rc.amount:
+            pdoc.update({'amount': rc.amount})
+        else:
+            pdoc.update({'amount': 'tbd'})
+        if rc.authors:
+            if isinstance(rc.authors, str):
+                pdoc.update({'authors': [rc.authors]})
+            else:
+                pdoc.update({'authors': rc.authors})
+        else:
+            pdoc.update({'authors': 'tbd'})
         pdoc.update({
-            'begin_date': now,
-            'log_url': '',
-            'name': rc.name,
-            'pi_id': rc.pi_id,
-            'lead': rc.lead,
+            'begin_day': rc.begin_day,
+            'begin_month': rc.begin_month,
+            'begin_year': rc.begin_year,
+            'call_for_proposals': rc.call_for_proposals,
+            # 'cpp_info':
         })
-        if rc.lead is "tbd":
-            pdoc.update({
-                'status': 'proposed'
-            })
+        if rc.currency:
+            pdoc.update({'currency': rc.currency})
         else:
-            pdoc.update({
-                'status': 'started'
-            })
-
-        if rc.description:
-            pdoc.update({
-                'description': rc.description,
-            })
-        if rc.grants:
-            if isinstance(rc.grants, str):
-                rc.grants = [rc.grants]
-            pdoc.update({'grants': rc.grants})
+            pdoc.update({'currency': 'USD'})
+        pdoc.update({'day': now.day
+                     'due_date': rc.due_date
+                     })
+        if rc.duration:
+            pdoc.update('duration': rc.duration)
         else:
-            pdoc.update({'grants': ["tbd"]})
-        if rc.group_members:
-            if isinstance(rc.group_members, str):
-                rc.group_members = [rc.group_members]
-            pdoc.update({'group_members': rc.group_members})
+            pdoc.update({'duration': 'tbd'})
+        pdoc.update({
+            'end_day': rc.end_day,
+            'end_month': rc.end_month,
+            'end_year': rc.end_year,
+            'funder': rc.funder,
+            'full': rc.full
+            'month': now.month # convert numeric to string
+            'notes': rc.notes
+        })
+        if rc.pi:
+            pdoc.update({'pi': rc.pi})
+        else:
+            pdoc.update({'pi': 'tbd'})
+        if rc.status:
```

Comment:
auto-set this to ``inprep``
remove from CLI

Reply:
are you referring to `pi` or `status`?

Reply:
comments always appear below the line they refer to....